src/pyscipopt/scip.pxi

```diff
@@ -2834,6 +2834,35 @@ cdef class Model:
         PY_SCIP_CALL(SCIPreleaseCons(self._scip, &scip_cons))
 
         return pyCons
+    
+    def addPiecewiseLinearCons(self, X, Y, a, b):
+        """add piecewise relation with multiple selection formulation
+
+            :param a: array with x-coordinates of the points in the piecewise linear relation
+            :param b: array with y-coordinate of the points in the piecewise linear relation
+        """
+        K = len(a)-1
```

Comment:
Should we maybe assert that lists a,b are of equal length?

Reply:
Good idea!